ui/webui/src/components/app.jsx

```diff
@@ -107,9 +112,31 @@ export const Application = () => {
         cockpit.location.go([newStep.id]);
     };
     const title = _("Anaconda Installer");
+    const prerelease = _("Pre-release");
 
     return (
-        <Page data-debug={conf.Anaconda.debug}>
+        <Page
+          data-debug={conf.Anaconda.debug}
+          additionalGroupedContent={
+              <PageSection variant={PageSectionVariants.light}>
+                  <Flex spaceItems={{ default: "spaceItemsSm" }} alignItems={{ default: "alignItemsCenter" }}>
+                      <TextContent>
+                          <Text component="h1">{title}</Text>
+                      </TextContent><br />
```

Comment:
Please don't use br for spacing. That's what Flex and other Layout components from PF4 are for. If this is not working as expected, you might need to wrap your Popver in a FlexItem. will check it interactively and propose a solution.